modules/api/src/main/java/org/eclipse/xpanse/api/controllers/OpenTofuMakerWebhookApi.java

```diff
@@ -49,6 +50,7 @@ public class OpenTofuMakerWebhookApi {
     @PostMapping(value = "${webhook.tofu-maker.deployCallbackUri}/{task_id}", produces =
             MediaType.APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.OK)
+    @GetCspMethodInfo(enabled = false, methodName = "getCspWithServiceId")
```

Comment:
if it is not enabled, why do we need the methodName? Comment applies to few other methods as well.  Please check all.